src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs

```diff
@@ -223,6 +229,30 @@ protected void VerifyCurrentSignature(Signature expectedSignature)
             Assert.Equal(expectedSignature, currentSignature);
         }
 
+        protected void VerifyCurrentSignature(string content)
```

Comment:
Did we have other tests using the other helper above? Should we be using that instead?